frontend/src/components/UserCard.js

```diff
@@ -30,7 +30,7 @@ export default class UserCard extends Component {
 
     const href = `https://opencollective.com/${user.username}`;
 
-    const roleLabel = formatAnchor(user.tier || user.role);
+    const roleLabel = formatAnchor(user.role || user.tier);
```

Comment:
Ah. That stuff is a mess. I don't have an immediate better fix for it. Tiers cause us more small bugs than anything else.